### ISSUE TITLE
State of Toolbar in context menu and View | Toolbar are not always synchronized

### ISSUE THREAD

**Issue by EugeneZelenko** [KLayout/klayout#1067]
When Toolbar is hidden via Toolbar in context menu, check mark is still in View | Toolbar menu. This problem exists in 0.27.9 and 0.26.4.

**Comment by klayoutmatthias** [comment]
You mean this checkbox is still set?

![image](https://user-images.githubusercontent.com/25720841/165397618-a9412963-a5b0-4f1b-ba77-793bf986cd59.png)

In my case it isn't (Ubuntu-20). What is the system you're on?

Matthias

**Comment by EugeneZelenko** [comment]
Yes. I use Ubuntu 18, your build for Ubuntu 18 (0.27.9) and custom build on Red Hat (0.26.4).

**Comment by klayoutmatthias** [comment]
I cannot see this issue on Windows too.

Maybe it is related to the Window Manager. What is the manager you are using?

Matthias

**Comment by EugeneZelenko** [comment]
Looks like it's GNOME:

`XDG_CURRENT_DESKTOP=ubuntu:GNOME`
`GDMSESSION=ubuntu`

Eugene.

**Comment by klayoutmatthias** [comment]
I'm having trouble seeing that issue. Is that still valid?

**Comment by EugeneZelenko** [comment]
Yes, problem is still there. I also tried 0.27.13.
